### DISCUSSION TITLE
Future of UnrealCLR

### DISCUSSION THREAD

**Discussion by ScottKane** [nxrighthere/UnrealCLR#168] [General]
I am just wondering what is planned for the future of UnrealCLR? Coverage of the Unreal APIs seems to have slowed down and updates look to mainly be for UE/.NET version upgrades. Would be nice to know if we will get UE5 support and if most of the existing UE APIs will be covered by then?

**Comment by nxrighthere** [comment]
> Would be nice to know if we will get UE5 support

It's supported since [May](https://github.com/nxrighthere/UnrealCLR/releases/tag/1.29).

> if most of the existing UE APIs will be covered by then?

What API you would like to see exposed?

> I am just wondering what is planned for the future of UnrealCLR?

All that is specified in the [roadmap](https://github.com/users/nxrighthere/projects/5?fullscreen=true).

> Coverage of the Unreal APIs seems to have slowed down and updates look to mainly be for UE/.NET version upgrades.

Most of my work is not reflected directly in repository updates. What you see in commits is just a small part of it.

**Comment by ScottKane** [comment]
I think exposing the Slate API and possibly Networking. I would say these are two of the biggest things stopping the framework from being able to create something end to end. Also I think it would be nice to have access to one of the particles API's. I didn't see that the framework already supports UE5 so apologies

**Reply by nxrighthere** [reply]
> I think exposing the Slate API and possibly Networking.

Both of these in one or another way are tied on https://github.com/nxrighthere/UnrealCLR/issues/61 and https://github.com/nxrighthere/UnrealCLR/issues/33, this is the only reason why it's not available yet. Please, see https://github.com/nxrighthere/UnrealCLR/discussions/171#discussioncomment-1135148.
